gallery/gallery/lib/l10n/intl_en_US.arb

```diff
@@ -13,7 +13,7 @@
       }
     }
   },
-  "signIn": "Sign In",
+  "signIn": "SIGN IN",
```

Comment:
Re-run the l10n script since you changed this

Reply:
Done